db/schema.rb

```diff
@@ -803,7 +805,6 @@
     t.integer "unspent_credits_count", default: 0, null: false
     t.datetime "updated_at", null: false
     t.string "url"
-    t.datetime "latest_article_updated_at"
```

Comment:
I reset my db and migrated and this was still showing up. Given we alphabetize our schema I think this ordering was committed somehow by mistake and we want this fix.